README.md

```diff
@@ -78,7 +78,7 @@ public interface Publisher<T> {
 
 | ID                        | Rule                                                                                                   |
 | ------------------------- | ------------------------------------------------------------------------------------------------------ |
-| <a name="1.1">1</a>       | The number of `onNext` signaled by a `Publisher` to a `Subscriber` MUST NOT exceed the cumulative demand that has been signaled via that `Subscriber`’s `Subscription` |
+| <a name="1.1">1</a>       | The number of `onNext` signals sent by a `Publisher` to a `Subscriber` MUST always be less than or equal to the elements requested by that `Subscriber`´s `Subscription` |

```

Comment:
Now looking at this rule, there is a slight imprecision here. "less than or equal to the elements requested" is a bit vague and can be interpreted as the last request(X) call. What about:

"The total number of `onNext` signals sent by a `Publisher` to a `Subscriber` until any point in time MUST always be less than or equal to the total elements requested by that `Subscriber`´s `Subscription until that time."

Maybe reworded, my point is to make it clear that "elements requested" is meant as an aggregate, not just the last request() call.